languages/java/oso/src/main/java/com/osohq/oso/Host.java

```diff
@@ -92,16 +92,13 @@ public Long cacheInstance(Object instance, Long id) throws Exceptions.OsoExcepti
      * Make an instance of a Java class from a {@code Map<String, Object>} of
      * fields.
      */
-    public Object makeInstance(String className, List<Object> initargs, long id)
-            throws Exceptions.OsoException
-    {
+    public Object makeInstance(String className, List<Object> initargs, long id) throws Exceptions.OsoException {
```

Comment:
Are these formatting changes due to version differences in rustfmt?

Reply:
We don't have a standardised java formatter, but apparently my IDE does.

Reply:
It would be good to pick one and standardize on it (not as part of this PR)